Makefile.plugins.mk

```diff
@@ -319,6 +318,7 @@ $(BUILD_DIR)/step1.rsrc: $(DPF_PATH)/distrho/src/DistrhoPluginAU.r $(BUILD_DIR)/
         -i $(DPF_PATH)/distrho/src/CoreAudio106/AudioUnits/AUPublic/OtherBases \
         -i $(DPF_PATH)/distrho/src/CoreAudio106/AudioUnits/AUPublic/Utility \
         -i $(DPF_PATH)/distrho/src/CoreAudio106/PublicUtility \
+        ${REZFLAGS} \
```

Comment:
I needed this extra flag so I can pass in "-isysroot /Developer/SDKs/MacOSX10.6.sdk"